File: problem_twenty_four.py
# L C M
import math
a  = int(input("Enter the numer:"))
b  = int(input("Enter the number:"))
def LCM(a,b):   
    if a > b:
        t = a
        a = b
        b = t
    lst = []    
    for i in range(a,1,-1):
        Z = b % i
        X = a % i
        if Z == 0 and X == 0:
            lst.append(i)
            k = b / i
            y = a / i
            b = k
            a = y
    lst.append(a)
    lst.append(b) 
    val = 1       
    for i in lst:
       val*=i 
    print("LCM = ",math.floor(val)) 
LCM(a,b)    

            

File: problem_twenty_four.js
// LCM
var a = 15
var b = 5
function myFunction(a,b){
    if(a>b){
        t = a
        a = b
        b = t
    }
    var lst =[]
    for(var i = a; i>1;i--){
        Z = b % i
        X = a % i
        if(Z== 0 && X == 0){
            lst.push(i)
            k = b / i
            y = a / i
            b = k
            a = y
        }
    }
    lst.push(a)
    lst.push(b)
    var value = 1
    for (var j =0; lst.length> j;j++){
        value *= lst[j]
    }
    console.log("LCM = ",value)
}
myFunction(a,b)